.github/workflows/ci.yml

```diff
@@ -83,11 +83,15 @@ jobs:
       - name: test npm
         if: matrix.kind == 'npm'
         run: ./gradlew testNpm
+      - name: Setup go
+        if: matrix.kind == 'shfmt'
+        uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
       - name: Install shfmt
         if: matrix.kind == 'shfmt'
         run: |
-          curl -sSfL "https://github.com/mvdan/sh/releases/download/v${{ matrix.shfmt-version }}/shfmt_v${{ matrix.shfmt-version }}_linux_amd64" -o /usr/local/bin/shfmt
-          chmod +x /usr/local/bin/shfmt
+          go install mvdan.cc/sh/v3/cmd/shfmt@${{ matrix.shfmt-version }}
```

Comment:
Does this mean `$GOPATH/bin` has been recognized now? What makes it workable?

Reply:
Yes. You can see in the [README for the setup-go action](https://github.com/actions/setup-go?tab=readme-ov-file#v3) that as of v3, the action:
>Adds `GOBIN` to the `PATH`

Reply:
Oooops, I missed you applied `actions/setup-go` in the step above.